bigbluebutton-html5/imports/ui/components/plugins-engine/loader/manager.tsx

```diff
@@ -32,6 +32,9 @@ const PluginLoaderManager = (props: PluginLoaderManagerProps) => {
     script.src = plugin.url;
     script.setAttribute('uuid', div.id);
     script.setAttribute('pluginName', plugin.name);
+    if (plugin.checksum) {
```

Comment:
Should we even allow plugins without checksum?

Reply:
I would keep it optional as it can facilitate for development.

Reply:
I agree